### DISCUSSION TITLE
Printing with Symfony 5.2

### DISCUSSION THREAD

**Discussion by plouha** [symfony/symfony#44559] [Q&A] [answered]
Hi everybody,          (sorry for my poor english ... I'm french)
I'm doing a selection of elements in my database with a findby.
I got the result perfectly on my screen.
But my problem is ... **how to put a button "PRINT" on the screen to have this selection result in a file .PDF and regsiter it on my computer.**
I loocked everywhere but did'nt found any doc about it.
Does someone can help on this subject ?
Thank you for any answer (... if possible in french)
Best regards

**Comment by GromNaN** [comment, marked as answer]
Hello @plouha. Symfony does not provide such feature, but tools to build your own solution. You can use Twig to render printable HTML. https://www.alsacreations.com/tuto/lire/586-feuille-style-css-print-impression.html

**Reply by plouha** [reply]
Never heard about that before. I'm going to look at it.
Thank's a lot
